pkg/scheduler/apis/config/validation/validation_pluginargs.go

```diff
@@ -269,6 +269,18 @@ func validateResources(resources []config.ResourceSpec, p *field.Path) field.Err
 	return allErrs
 }
 
+// ValidateNodeResourcesBalancedAllocationArgs validates that NodeResourcesBalancedAllocationArgs are correct.
+func ValidateNodeResourcesBalancedAllocationArgs(path *field.Path, args *config.NodeResourcesBalancedAllocationArgs) error {
+	var allErrs field.ErrorList
+	for i, resource := range args.Resources {
```

Comment:
do you want to detect duplicated resource items?

Reply:
support it was a repeated but valid configuration,  do you think the scheduler will fail in this case?

Reply:
It wouldn't necessarily fail, but it doesn't make sense. Better validate it.

Reply:
Done